docs/library/array.rst

```diff
@@ -19,6 +19,10 @@ Classes
     array are given by *iterable*. If it is not provided, an empty
     array is created.
 
+    Array objects also implement the buffer protocol, meaning the contents can
```

Comment:
Reading this again, the word "also" is out of place here.  I suggest "In addition to the methods below, array objects also ...".

Reply:
I've made an edit along these lines. I think this is why I originally decided to put this note at the end, because the array is primarily an array of typed objects and only secondarily a flat buffer. The version in d3932ad3f17c9a000d867f6ecb1d4bdb7c6925ca may be less confusing for someone who is new to the `array` module. However I don't feel very strongly either way.